corehq/apps/translations/tasks.py

```diff
@@ -22,13 +22,12 @@
 
 @task
 def delete_resources_on_transifex(domain, data, email):
-    version = data.get('version')
     transifex = Transifex(domain,
-                          data.get('app_id'),
-                          data.get('target_lang') or data.get('source_lang'),
-                          data.get('transifex_project_slug'),
-                          version,
-                          use_version_postfix='yes' in data['use_version_postfix'])
+                          app_id=None,
+                          source_lang=None,
+                          project_slug=data.get('transifex_project_slug'),
+                          version=None,
+                          use_version_postfix=[])
```

Comment:
I think I was mistaken about the intent of the "delete" which according to our documentation will:
> Delete all resources (including translations) **for a version**

In my local testing I found it would seemingly always delete **all** resources, but admittedly I may not have thoroughly checked with versioning.

Reply:
hmm would deleting by versions work by choosing both a version from the dropdown menu and checking "Use version postfix"? And a bigger question given the time constraints is is this an important enough feature to block this PR from merging, given that there's a workaround (manually deleting desired resources on Transifex)?

Reply:
thanks for the insight Manish! I can do a bit of testing with the versioning and restore it after my demo during the tech meeting.

Reply:
restored this function - I should've verified beforehand but I did confirm that it's able to delete specific version of an app, if you've already created resources for that version